src/framerate_gui.cpp

```diff
@@ -52,30 +52,30 @@ namespace {
 		static const TimingMeasurement INVALID_DURATION = UINT64_MAX;
 
 		/** Time spent processing each cycle of the performance element, circular buffer */
-		TimingMeasurement durations[NUM_FRAMERATE_POINTS];
+		std::array<TimingMeasurement, NUM_FRAMERATE_POINTS> durations;
```

Comment:
Should these have `{}` at the end to initialise them?

Reply:
Nice catch!